nservicebus/sql-persistence/saga.md

```diff
@@ -55,6 +55,36 @@ snippet: SqlPersistenceSaga
 snippet: SqlPersistenceSqlSaga
 
 
+## Requirement for the SqlSagaAttribute
+
+When looking at a standards Saga's `ConfigureHowToFindSaga` it is logical to come to the conclusion that it should be possible to infer the Correlation Id and hence remove the requirement for defining a `[SqlSagaAttribute]`.
```

Comment:
This sentence seems off and I think it's because it;s avoiding saying "you might think...".

> In a standard Saga, the Correlation Id is configured in the `ConfigureHowToFindSaga` method. When using the Sql Persistence, each saga must be decorated with a `[SqlSagaAttribute]` which defines the Correlation Id again. There are a few good reasons for that.

And then you can close with a statement about defining it twice is duplication and the `SqlSaga<>` base class was introduced to avoid that duplication.